onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -362,13 +349,8 @@ void FillInputBlob(InferenceEngine::Blob::Ptr& inputBlob, size_t request_id, siz
   auto input_data = minputHolder.as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
   size_t input_data_size = inputBlob->byteSize();
 
-#if defined OPENVINO_2020_3
-  ORT_UNUSED_PARAMETER(input_name);
-  const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context.input_indexes[request_id]);
-#else
   ORT_UNUSED_PARAMETER(request_id);
```

Comment:
If request_id is not going to be used anywhere in the function, then we should remove it completely from the function parameter list.

Reply:
@smkarlap Done Manohar. Thanks for finding this. request_id is now completely removed.

**commit Message:**
request_id parameter was being used with ov_2020.3
release. Starting from 2020.4 OV release, input_name
parameter is being used instead to get the
KernelContext_GetInput.

so we went ahead and removed it along with it's references.